spotlessSelf.gradle

```diff
@@ -6,18 +6,19 @@ plugins {
 
 repositories { jcenter() }
 spotless {
+	def noInternalDepsClosure = {
+		if (it.contains('import org.gradle.internal.') &&
+		!it.contains('def noInternalDepsClosure')) {
```

Comment:
Oh, I'm probably mistaken!

I can see that if this `!it.contains('def noInternalDepsClosure')` condition is _not_ in place, then running `spotlessSelf.gradle` on itself would cause an `AssertionError`.

Never mind! :stuck_out_tongue_closed_eyes: